src/lib/types/EmbeddingEndpoints.ts

```diff
@@ -0,0 +1,39 @@
+import { z } from "zod";
```

Comment:
small nit: this file should be named from `EmbeddingEndpoints.ts` to `EmbeddingEndpoint.ts` (without the plural `s`)